string object
---
float32[] position